src/files-card-body.scss

```diff
@@ -0,0 +1,195 @@
+        cursor: pointer;
+    }
+
+    &.view-details .item-name a::before {
+        display: inline-block;
+        margin-inline-end: var(--pf-v5-global--spacer--sm);
+        // FIXME: Figure out why there's an offset
+        transform: translateY(3px);
+    }
+
+    &.view-grid .item-name a {
+        display: block;
+
+        &::before {
+            display: block;
+            margin-inline: auto;
+            margin-block-end: var(--pf-v5-global--spacer--sm);
+        }
+    }
+
+    &.view-details {
+        --icon-size: 16px;
+        inline-size: calc(100% - var(--pf-v5-global--spacer--md));
+        margin-block: var(--pf-v5-global--spacer--sm);
+        margin-inline: auto;
+
+        tbody tr {
+            border-block: 1px solid var(--pf-v5-global--BorderColor--100);
+        }
+
+        tbody > tr:hover {
+            background-color: var(--pf-v5-global--BackgroundColor--200);
+        }
+
+        th, td {
+            padding: var(--pf-v5-global--spacer--sm);
+            border-block-start: none;
+        }
+
+        .col-size, .item-size,
+        .col-date, .item-date {
+            inline-size: 12ch;
+            text-align: end;
+        }
+
+        // Remove the extra padding on the end of the column and button, as the icon has padding already
+        .col-size, .col-date {
+            &, .pf-v5-c-table__button {
+                padding-inline-end: 0;
+            }
+
+            .pf-v5-c-table__button {
+                inline-size: 100%;
+                justify-content: end;
+            }
+        }
+    }
+
+    &.view-grid {
+        --icon-size: 64px;
+        display: contents;
+
+        thead {
+            display: none;
+        }
+
+        tbody {
+            align-items: start;
+            display: grid;
+            // As we're using justify-content: space-between, this is the minimum gap horizontally;
+            gap: var(--pf-v5-global--spacer--sm);
+            grid-template-columns: repeat(auto-fill, minmax(8rem,1fr));
+            justify-content: space-between;
+            margin: var(--pf-v5-global--spacer--sm);
+        }
+
+        tr {
+            display: block;
+            // Override default PF padding
+            padding: 0;
+
+            td {
+                display: block;
+                text-align: center;
+                word-break: break-all;
+                // Override default PF padding
+                padding: 0;
+
+                a {
+                    padding: var(--pf-v5-global--spacer--sm);
+                }
+
+                &:not(:last-child) a {
+                    padding-block-end: 0;
+                }
+
+                + td {
+                    padding-block-end: var(--pf-v5-global--spacer--sm);
+                }
+            }
+
+            .item-num {
+                display: none;
+            }
+
+            .item-size,
+            .item-date {
```

Comment:
I think the inclusion of `.item-date` here is probably accidental since you `display: none` it, just below?

Reply:
Howso, we want the font-size and color to be the same?

Reply:
But we don't show the date...